aixplain/modules/finetune.py

```diff
@@ -87,6 +88,7 @@ def __init__(
         self.cost = cost
         self.train_percentage = train_percentage
         self.dev_percentage = dev_percentage
+        self.parameters = parameters
```

Comment:
Finetune should have a Optional[Hyperparameter] and Optional[Peft] attributes, not?

This "parameters" seems wrong.

Reply:
The Optional[Hyperparameter] and Optional[Peft] parameters are in the [FinetuneFactory,create method](https://github.com/aixplain/aiXplain/blob/M-5470304031-addFinetunePromptParameters/aixplain/factories/finetune_factory/__init__.py#L102). The above class is only created by FinetuneFactory and it is used to call the [sdk/finetune endpoint](https://github.com/aixplain/aiXplain/blob/M-5470304031-addFinetunePromptParameters/aixplain/modules/finetune.py#L123) from backend, so we can pass directly the parameters dictionary.